src/host/VtApiRoutines.cpp

```diff
@@ -424,15 +424,15 @@ void VtApiRoutines::GetLargestConsoleWindowSizeImpl(const SCREEN_INFORMATION& co
 
 [[nodiscard]] HRESULT VtApiRoutines::SetConsoleWindowInfoImpl(SCREEN_INFORMATION& context,
                                                               const bool isAbsolute,
-                                                              const SMALL_RECT& windowRect) noexcept
+                                                              const til::inclusive_rect& windowRect) noexcept
 {
-    (void)m_pVtEngine->_ResizeWindow(windowRect.Right - windowRect.Left, windowRect.Bottom - windowRect.Top);
+    (void)m_pVtEngine->_ResizeWindow(windowRect.Right - windowRect.Left + 1, windowRect.Bottom - windowRect.Top + 1);

```

Comment:
is this a bug fix?

Reply:
Yes. From what I can tell, `SetConsoleWindowInfoImpl` truly is being called with an inclusive rectangle, just like the documentation for `SetConsoleWindowInfo` says.